scripts/bundleDaemon.sh

```diff
@@ -1,17 +1,15 @@
 #!/bin/bash
-# Usage: bash bundleDaemon.sh <daemonVersion> [githubAuthHeader]
```

Comment:
Can we add something here explaining that `$GITHUB_TOKEN` is a requirement if someone wanted to use this script locally ?

Reply:
It's not a requirement though 😛 I've successfully run the script locally without it.